src/aktualizr_secondary/secondary_tcp_server.cc

```diff
@@ -110,6 +110,7 @@ bool SecondaryTcpServer::HandleOneConnection(int socket) {
         auto r = resp->manifestResp();
         r->manifest.present = manifest_PR_json;
         SetString(&r->manifest.choice.json, manifest);  // NOLINT
+        LOG_TRACE << "Manifest : \n" << manifest;
```

Comment:
The manifest was print frequently in my test because we always keep connect/disconnect, maybe annoying. Is it make sense to remove this line ? Or maybe only print it once until next reboot or installation ?

Reply:
I think this is fine for now at trace level. That's expected to be very verbose, and the manifest is useful.